src/main/resources/db/migration/0250/V272__ParticipantProjectSpecies.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE accelerator.participant_project_species (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    project_id BIGINT REFERENCES public.projects ON DELETE CASCADE,
+    species_id BIGINT REFERENCES public.species ON DELETE CASCADE,
+    submission_status_id INTEGER NOT NULL REFERENCES accelerator.submission_statuses,
+    feedback TEXT,
+    rationale TEXT,
+
+    UNIQUE (project_id, species_id)
+);
+
+CREATE INDEX ON accelerator.participant_project_species (project_id);
```

Comment:
The index created by the unique constraint will already serve double duty as an index on project ID, but there needs to be an index on species ID.